### PR TITLE
Font/Art Specifications Update

### PR DESCRIPTION
**Pull request by lwtchu**
Updated some verbiage for more accuracy and clarity

### CODE REVIEW

File: scripts/test/cod_documentation/style-guide.scss [hunk 1]
```diff
@@ -751,14 +753,16 @@
 
 /**
 @Section
-  Image Layout
+  Image Layout &amp; Settings
```
Comment (philschatz) [inline scripts/test/cod_documentation/style-guide.scss]: Is `&amp;` needed or would `&` work as well?

Comment (lwtchu) [reply scripts/test/cod_documentation/style-guide.scss]: I'm not..sure. Will `&` break it? I am only 50% sure about what I'm doing in here :joy:.

Comment (tomjw64) [reply scripts/test/cod_documentation/style-guide.scss]: @lwtchu Looks like you actually need to use `&`

Comment (lwtchu) [reply scripts/test/cod_documentation/style-guide.scss]: Okay ..how do I undo this D:

**Comment by lwtchu** [comment]
Okay I THINK i made the changes. Sorry for the GitHub incompetence 🙏

**Pull request merged**